views.py

```diff
@@ -107,3 +107,8 @@ def tofu(request):
     'View for tofu page.'
     return render(request, 'tofu.html', {'title' : 'Tofu',
                                          'tagline' : 'Not just for eating'})
+
+def about(request):
+    'View function for about page of site.'
```

Comment:
The docstring should be in triple quotes.  I'm pretty sure this got copied from my bad example.  😅